packages/edit-site/src/components/block-editor/index.js

```diff
@@ -51,11 +51,12 @@ export default function BlockEditor() {
 		'postType',
 		settings.templateType
 	);
-	const setActiveTemplateId = useCallback(
-		( newTemplateId ) =>
+	const setActivePageAndTemplateId = useCallback(
+		( { page, templateId } ) =>
 			setSettings( ( prevSettings ) => ( {
 				...prevSettings,
-				templateId: newTemplateId,
+				page,
```

Comment:
I found it had to understand what exactly the `page` is here. I know this existed before, but some types might be helpful.

Reply:
It will make sense once it's in the store.